src/components/symptom-shark/presentational/MonthReport/MonthReport.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useContext } from "react";
+import "./MonthReport.css";
+import NotesTimeline from "../NotesTimeline/NotesTimeline";
+import BulletChart from "../BulletChart/BulletChart";
+import OverallExperienceChart from "../OverallExperienceChart/OverallExperienceChart";
+import SymptomSeverityChart from "../SymptomSeverityChart/SymptomSeverityChart";
+import { getMonthName } from "../../../../helpers/date-helpers";
+import { DateRangeContext, TextBlock } from "../../../presentational";
+import { SymptomSharkVisualizationContext } from "../../container/VisualizationCoordinator/VisualizationCoordinator";
+import { startOfMonth } from "date-fns";
+
+export interface SymptomSharkMonthReportProps {
+    includeNotes: boolean;
+    includeDailyOverallFeeling: boolean;
+    productLogo?: string;
+    productName?: string;
+    productUrl?: string;
+    intervalStart?: Date;
+    innerRef?: React.Ref<HTMLTableElement>;
+}
+
+export default function (props: SymptomSharkMonthReportProps) {
+    let visualizationContext = useContext(SymptomSharkVisualizationContext);
+    if (!visualizationContext) {
+        return <TextBlock innerRef={props.innerRef}>Error: Month report must be used inside a Symptom Shark Visualization Coordinator.</TextBlock>
+    }
+
+    let dateRangeContext = useContext(DateRangeContext);
+    let intervalStart = props.intervalStart || startOfMonth(new Date());
+    if (dateRangeContext) {
+        intervalStart = dateRangeContext.intervalStart;
+    }
+
+    if (!visualizationContext.symptoms.length && !visualizationContext.treatments.length) {
+        return null;
+    }
+
+    return <table ref={props.innerRef} className="ss-month-report" cellPadding="0" cellSpacing="0">
```

Comment:
I don't mind the use of tables here, and I'm not sure if this was the reason, but I just wanted to mention that I had decent luck using `display: flex` on the asthma provider report, if you'd prefer to use CSS over tables.

Reply:
This is like 2-3 year old code now just porting to this library, so just gonna leave it alone